admin/includes/languages/english/products_to_categories.php

```diff
@@ -14,13 +14,11 @@
 
 define('TABLE_HEADING_PRODUCTS_ID', 'Prod ID');
 define('TABLE_HEADING_PRODUCT', 'Product Name');
-define('TABLE_HEADING_MODEL', 'Model');

```

Comment:
You've removed this, but not re-defined it elsewhere, yet it's used in `category_product_listing.php` and `downloads_manager.php`